src/test/java/net/sf/jabref/logic/formatter/bibtexfields/NormalizePagesFormatterTest.java

```diff
@@ -36,6 +36,11 @@ public void ignoreWhitespaceInPageNumbers() {
     }
 
     @Test
+    public void removeWhitespaceForSinglePageNumber() {
+        expectCorrect("   1  ", "1");
```

Comment:
Yes they do. I added this test also to make sure that single pages are detected.